test/tools/yulInterpreter/EWasmBuiltinInterpreter.cpp

```diff
@@ -144,7 +152,7 @@ u256 EWasmBuiltinInterpreter::evalBuiltin(YulString _fun, vector<u256> const& _a
 		// TODO this does not read the address, but is consistent with
 		// EVM interpreter implementation.
 		// If we take the address into account, this needs to use readAddress.
-		return writeU128(arg[0], m_state.balance);
+		return writeU128(arg[1], m_state.balance);
```

Comment:
This is a bugfix since the balance pointer is the second argument.

Reply:
Yes it is. Do you want me to move it to separated PR?

Reply:
No, it was just a review confirmation that this is correct.